EIPS/eip-5375.md

```diff
@@ -0,0 +1,293 @@
+---
+eip: 5375
+title: NFT Authorship
```

Comment:
Could you make a more specific title? This is a bit short IMO.

Reply:
Added new title in 4785b643599f54b0bf5c5e2483c8772f6142add7, waiting for the green light before resolving.